framework/util/to_string.h

```diff
@@ -28,10 +28,13 @@
 
```

Comment:
Remaining to_string code is used by DX12 ascii consumer and in some logging code that wants a pretty representation of a value it wants to show to the user. There may still be dead code lurking that we can continue to cull when we spot it later.